### PR TITLE
[UT] fix unstable UT BDBEnvironmentTest#testRollbackExceptionOnSetupCluster

### PR DESCRIPTION
**Pull request by padmejin**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [x] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

This test case tries to simulate an unexpected scenario where a master was down before it got way ahead of all followers. Normally BDB will cause a `RollbackException`, which we should handle and turn to a `JournalException`. But there is a slight chance that BDB may handle it well and nothing goes wrong. Either way, we think it's reached our expectation.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by gengjun-git — DISMISSED** [review]

File: fe/fe-core/src/test/java/com/starrocks/journal/bdbje/BDBEnvironmentTest.java [hunk 1]
```diff
@@ -278,8 +278,13 @@ public void testDeleteDb() throws Exception {
 
     /**
      * see https://github.com/StarRocks/starrocks/issues/4977
+     *
+     * This test case tries to simulate an unexpected scenario where a master was down before it got way ahead of all
```
Comment (chaoyli) [inline fe/fe-core/src/test/java/com/starrocks/journal/bdbje/BDBEnvironmentTest.java:282]:
```suggestion
     use leader to replace master
```

Comment (padmejin) [reply fe/fe-core/src/test/java/com/starrocks/journal/bdbje/BDBEnvironmentTest.java:282]: Good point.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by chaoyli — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**